data-raw/dummy_anchored.R

```diff
@@ -0,0 +1,64 @@
+#### create anchored example datasets ####
+
+devtools::load_all()
+library(flexsurv)
+set.seed(2024)
+
+# create adsl_twt
+adsl <- read.csv(system.file("extdata", "adsl.csv",
+  package = "maicplus",
+  mustWork = TRUE
+))
+adsl$USUBJID <- paste0("xx", adsl$USUBJID)
+adsl2 <- adsl
```

Comment:
Do we need to set the ARM here?
```suggestion
adsl2 <- adsl
adsl2$ARM <- "C"
```

Reply:
Well.. adtte has arm name "A". Code probably wants to simulate control arm "C"